build.csx

```diff
@@ -0,0 +1,217 @@
+#load "packages/simple-targets-csx.5.2.0/simple-targets.csx"
+
+#r "System.Xml.Linq"
+
+//using System.Runtime.Serialization.Json;
+//using System.Xml;
+//using System.Xml.Linq;
+using static SimpleTargets;
+
+static readonly var  solutionName = "FakeItEasy";
+
+static readonly var solution = "./" + solutionName + ".sln";
+static readonly var packagesDir = Path.GetFullPath("packages");

```

Comment:
Should this be `packagesDirectory`, for consistency with `logsDirectory`, `outputDirectory` and `testsDirectory`?